packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -228,6 +233,19 @@ function toTree(vnode) {
         rendered: childrenToTree(node.child),
       };
     }
+    case FiberTags.Suspense: {
+      return {
+        nodeType: 'suspense',
```

Comment:
hmm, is this nodeType needed? it should probably be `function`, no?

Reply:
Frankly speaking I'm not quite certain what `nodeType` it should be. I just think (probably I'm wrong) it's not a functional component so it couldn't be `function` and should be other type. It seems that `nodeType` is only used in enzyme internally so maybe it's OK to just have it `function`, though?

Reply:
unless there's a reason for it to be different, I think it's fine to make it appear to be a functional component.

Reply:
Fixed in 173001f .